### DISCUSSION TITLE
Bar graph Seizure Probability MainActivity2

### DISCUSSION THREAD

**Discussion by AroonPro** [OpenSeizureDetector/Android_Pebble_SD#182] [05_Seizure Detection Algorithms]
@jones139 Is the bar graph Seizure Probability being forgotten in MainActivity2 or left out because unnecessary?

**Comment by AroonPro** [comment]
Added iin @AroonPro Android_Pebble_SD v5.0.0c

**Comment by jones139** [comment]
Will change this to an issue and fix it, thanks.

**Comment by jones139** [comment]
Included in V4.2.8
